doc/user/options.md

```diff
@@ -64,7 +64,7 @@ switches specific to TruffleRuby.
 TruffleRuby switches:
   -Xlog=severe,warning,performance,info,config,fine,finer,finest
                   set the TruffleRuby logging level
-  -Xoptions       print available TrufleRuby options
+  -Xoptions       print available TruffleRuby options
```

Comment:
The same typo exists in src/main/java/org/truffleruby/options/CommandLineParser.java:
https://github.com/graalvm/truffleruby/blob/c7e69aeb7b9c324eeee5377fa15ac5c245fa609e/src/main/java/org/truffleruby/options/CommandLineParser.java#L600